tests/modules/media_avportal_mock/responses/resources/I-053547.json

```diff
@@ -207,7 +207,7 @@
           }
         },
         "titles_json": {
-          "FR": "Launch of the FP7: Clip 1 \"Ensemble\""
+          "FR": "<strong>Launch</strong> of the FP7<hr>: Clip 1 \"Ensemble\""
```

Comment:
Let's add the test coverage to avoid double encoding here. Discussed with @sergepavle .